core/tests/protractor_desktop/adminTabFeatures.js

```diff
@@ -49,6 +49,27 @@ describe('Admin Tab', function() {
     adminPage.updateRole('collectionEditor1', 'collection editor');
     adminPage.getUsersAsssignedToRole('collection editor');
     adminPage.expectUsernamesToMatch(['collectionEditor1']);
+    users.logout();
+  });
+
+  it('should run,verify and stop one-off jobs', function() {
+    users.createAndLoginAdminUser('adminA@adminTab.com', 'alphaMan');
+    adminPage.getJobsTab();
+
+    adminPage.startOneOffJob('FeedbackThreadCacheOneOffJob');
```

Comment:
Why these jobs in particular, or is it arbitrary?

Reply:
They are arbitrary

Reply:
Can you add a comment to that effect?

Reply:
OK Done.